### PR TITLE
Update svelte example

### LINKED ISSUE #3501: Svelte example does not work
**core/examples/luigi-example-svelte** does not work anymore.
In addition please update luigi dependencies.

### PR DESCRIPTION
**Pull request by ndricimrr**
**Description**

Changes proposed in this pull request:

- Packages updated but still need to migrate to major svelte version. 

**Related issue(s)**

Fixes #3501

### CODE REVIEW

File: core/examples/luigi-example-svelte/package.json [hunk 1]
```diff
@@ -3,29 +3,29 @@
...
-    "style-loader": "^3.3.1",
-    "svelte": "^3.55.0",
-    "svelte-loader": "3.1.4",
-    "webpack": "^5.75.0",
-    "webpack-cli": "^5.0.1",
-    "concurrently": "^7.6.0",
-    "local-web-server": "^5.2.1"
+    "css-loader": "6.8.1",
+    "mini-css-extract-plugin": "^2.7.6",
+    "node-sass": "^9.0.0",
```
[the remark this answers is not in the document]
Comment (ndricimrr) [reply core/examples/luigi-example-svelte/package.json]: true, removed 👍

**Comment by camelCaseChris** [comment]
Looks good to me. I tried the updated example locally, it worked fine.

Just one glitch - not sure if you care about it. When running the `dev` script for the very first time, `serve:luigi` and `serve:microfrontend` exit with this:
```
[1] npm run serve:luigi exited with code 1
[2] The system cannot find the path specified.
[2] npm run serve:microfrontend exited with code 1
```
When running `dev` a second time, everything works.

From what I could see, the problem is that `serve:luigi` and `serve:microfrontend` start before `build:dev` is finished and they cannot handle the `public` folder not being there yet. You can simulate this by deleting the `public` folder and then running `dev` again.

**Comment by ndricimrr** [comment]
@camelCaseChris Good point. Adjusted as necessary.

**Review by camelCaseChris — APPROVED** [review]
Looks good to me.

**Pull request merged**